src/Components/WebAssembly/WebAssembly/src/HotReload/WebAssemblyHotReload.cs

```diff
@@ -50,14 +45,22 @@ internal static async Task InitializeAsync()
     [JSInvokable(nameof(ApplyHotReloadDelta))]
     public static void ApplyHotReloadDelta(string moduleIdString, byte[] metadataDelta, byte[] ilDelta, byte[] pdbBytes)
     {
+        // Analyzer has a bug where it doesn't handle ConditionalAttribute: https://github.com/dotnet/roslyn/issues/63464
+#pragma warning disable IDE0200 // Remove unnecessary lambda expression
+        if (_hotReloadAgent == null)
```

Comment:
Shouldn't this be `Interlocked.CompareExchange` to avoid multiple threads ending up with different instances?

Reply:
I think that if it has multiple initialization, it is not an issue for the runtime.
But I will change as you suggested to avoid any issue.